documentation/user_docs/docs/manual/Input_file_formats.rst

```diff
@@ -103,15 +206,51 @@ The SPE file contains the intensity and estimated standard deviation on those in
    :
 
 
-Here nd is the number of detectors, ne is the number of energy bins, phi contains scattering angles that are now ignored by all applications as well as Horace (set to 1,2,3...(nd+1)), en contains the energy transfer bin boundaries, and S and ERR contain the signal and standard error on the signal for each detecetor in turn.
+Here ``nd`` is the number of detectors, ``ne`` is the number of energy bins,
+``phi`` contains scattering angles, ``en`` contains the energy transfer bin
+boundaries, and ``S`` and ``ERR`` contain the signal and standard error on the
+signal for each detecetor in turn.
+
+.. warning::
+
+   The values in ``phi`` are conventionally ignored by neutron analysis
+   applications, including Horace, by default they are set to ``1, 2 ,3
+   ... (nd+1)``
+
+
+.. note::
+
+   On the first line, ``nd`` and ``ne`` need only to be separated by white
+   space.
+
+   In the blocks containing the signal and error the format is strongly
+   prescribed:
 
-On the first line, nd and ne need only to be separated by white space. In the blocks containing the signal and error the format is highly prescribed: each line must contain 8 real numbers, apart from the last line in each block, and each number must occupy a field of precisely 10 spaces. No white space is necessary. This is a frequent source of problem when writing the files. It is strongly recommended that you do not try to create your own SPE format files.
+   - each line must contain 8 real numbers, apart from the last line in each
+     block
 
+   - each number must occupy a field of precisely 10 spaces.
 
-PAR file format
-***************
+   - No white space is necessary. (This is a frequent source of problem when
```

Comment:
this reads very strangely. Not necessary implies you can take liberties with the formatting. By contrast the rigid format described in old line 108 means you must pad with spaces or zeros to the right field width. This needs at least checking